.github/workflows/build.yml

```diff
@@ -43,8 +43,8 @@ jobs:
           git checkout ${branch_name}
       - name: check for dependency branches and install if needed
         run: |
-          sh install_dependency_branch.sh layered_config_tree ${branch_name}
-          sh install_dependency_branch.sh vivarium ${branch_name}
+          sh ./.github/install_dependency_branch.sh layered_config_tree ${branch_name}
```

Comment:
Whoa, I don't think I've ever in my life used the .github/. What's the purpose of this?

Reply:
Its the folder path